docs/guides/esp32/ota.md

```diff
@@ -35,11 +35,24 @@ control list.
 
 ### Using Console
 
-After commissioning is successful, press Enter in requestor device console and
-type below query.
+After commissioning is successful, read the default-ota-providers list of
+requestor.
 
 ```
->matter ota query 1 <PROVIDER NODE ID> 0
+./out/debug/chip-tool otasoftwareupdaterequestor read default-otaproviders <REQUESTOR NODE ID> 0
```

Comment:
Typo: default-ota-providers

Reply:
It's not the typo. `default-otaproviders` is expected in command. 
I did write `default-ota-providers` in the instructions because for `default-otaproviders` spell check CI was failing.

Reply:
Ohh. I was then referring an older version of chip-tool that has `default-ota-providers`